arbor/mechcat.cpp

```diff
@@ -465,6 +476,7 @@ struct catalogue_state {
                 }
                 std::swap(new_rebind, over.ion_rebind);
             }
+            if (deriv.user_seed >= 0) over.user_seed = deriv.user_seed;
```

Comment:
In the user docs it states that the user seed has to be "positive": should it be "non-negative"?

Reply:
yep, non-negative. Point is moot now, since I removed the seed parameter.